backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1758,3 +1758,75 @@ databaseChangeLog:
             columns:
               - column:
                   name: test_result_delivery
+  - changeSet:
+      id: add-spring-session-jdbc-support
+      author: emma@skylight.digital
+      comment: Add a table to back spring sessions.
+      changes: 
+        - createTable:
+              tableName: spring_session
+              remarks: Data store for user sessions, as created by SpringSession.
+              columns:
+                - column:
+                    name: primary_id
+                    type: *idtype
+                    remarks: The immutable internal key to identify this session.
+                    constraints:
+                      nullable: false
```

Comment:
Isn't this the primary key?

Reply:
Yes, good call - updated. (I also added updated the rest of this file to match that postgres example I linked above, which included adding some indexes and more primary/foreign keys)